src/collection/layout.js

```diff
@@ -74,6 +88,26 @@ var elesfn = ({
     } else {
       nodes.positions( fn );
 
+      if ( options.spacingFactor && options.spacingFactor !== 1){
```

Comment:
Done. Right now `var spacing = Math.abs(options.spacingFactor)` is outside the function so that it doesn't get called *n* times in the no-animation case. It could also be moved inside the function which would make everything more concise (and would be one less argument that needs to be passed).